content/Community/contact.en.md

```diff
@@ -8,27 +8,37 @@ author: "Johan Ljunggren"
 
 #### #DSC channel
 
-The community gathers in the PowerShell #DSC channel which is an open
-forum for all DSC related questions, discussions and collaboration.
+The community gathers in the Virtual PowerShell User Group #DSC channel
+which is an open forum for all DSC related questions, discussions and
+collaboration.
 
 The PowerShell channels is bridged between Discord and Slack, so you
 can use either respectively app or website.
 
+There are not just the #DSC channel, there are a lot of other discussions
+around PowerShell in various channels. On top of the #DSC channel you can
+join any other channel as well.
+
+You can read more about the Virtual PowerShell User Group and how to
+connect at the site [poshcode.org](http://poshcode.org/). Below are the
+two most popular ways to connect.
+
 >*Tagging a specific user so they get notified works best in the app
 >that the user uses. Most maintainers and committee members has joined
 >through Slack.*
 
-There are not just the #DSC channel, there are a lot of other discussions
-around PowerShell in various channels. On top of the #DSC channel you can
-join any other channel as well.
+_If the links does not work then [poshcode.org](http://poshcode.org/)_
```

Comment:
"if the links *do* not" (plural)

Reply:
Done. Always having problem with this too. :)